usr/tests/test-malloc-mt.c

```diff
@@ -10,11 +10,11 @@
 #endif
 
 #ifndef NUM_ITER
-#define NUM_ITER    10000
+#define NUM_ITER    1000
```

Comment:
Why decrease this number?

Reply:
Just so it runs fine with less than 2 GB of memory